wrappers/canu/README.md

```diff
@@ -0,0 +1,71 @@
+# Documentation
+
+This wrapper makes long reads correction and/or assembly using Canu.
+
+This wrapper takes fastq or fasta file. Using `corrected` or `trimmed` step option,
+the output is corrected reads in a fasta.gz file. Without step option, the output
+is the assembly in a fasta file.
+
+**Required input:**
+
+- Long reads file (FASTQ/FASTA format)
+
+**Required output:**
+
+- Assembly result in fasta or corrected reads in fasta.gz.
+- The `canu{step}.done` file needed to let snakemake wait Canu.
```

Comment:
'...file is a trigger file to let Snakemake knows that canu computation is over'